templates/config/manifests/bases/clusterserviceversion.yaml.tpl

```diff
@@ -0,0 +1,73 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    category: "Cloud Provider"
+    alm-examples: '[]'
+    capabilities: {{.Annotations.CapabilityLevel}}
+    operatorframework.io/suggested-namespace: "ack-system"
+    repository: {{.Annotations.Repository}}
+    containerImage: {{.Annotations.ContainerImage}}:{{.ServiceIDClean}}-v{{.Version}}
+    description: {{.Annotations.ShortDescription}}
+    createdAt: {{.CreatedAt}}
+    support: {{.Annotations.Support}}
+    certified: {{.Annotations.IsCertified}}
+  name: ack-{{.ServiceIDClean }}-controller.v0.0.0
```

Comment:
is the `v0.0.0` suffix hard-coded here or should it be templatized like above on line 10?

Reply:
I'm not sure that it _needs_ to be `v0.0.0` in order to be parsed by operator-sdk, but operator-sdk does replace this with inputs that we pass it when creating the bundle. To that end, the Makefile's target `bundle` has our target version in place - and operator-sdk does the substitution.

Operator-SDK does **not** substitute the annotation value in line 10 as of this moment, so it's templated.